src/utils/navigationUtils.js

```diff
@@ -0,0 +1,43 @@
+/* globals window */
+export const isAbsoluteURL = (url) =>
+  !!(url.startsWith('http://') || url.startsWith('https://'))
+
+/**
+ * Determine whether a URL is for a different app from the current
+ * URL. In other words, navigating to the new URL cannot simply
+ * navigate within a single-page app.
+ * @param {String} newURL - The URL to test
+ * @return {Boolean} Whether the new URL is for another app
+ */
+export const isURLForDifferentApp = (newURL) => {
```

Comment:
Make some changes and simplify this:
* Don't consider absolute URLs a different app
* Remove any logic about search
* Remove any logic about homepage